extensions/ql-vscode/src/queries-panel/query-tree-data-provider.ts

```diff
@@ -68,8 +83,10 @@ export class QueryTreeDataProvider
     if (!item) {
       // We're at the root.
       return this.queryTreeItems;
-    } else {
+    } else if (item instanceof QueryTreeQueryItem) {
       return item.children;
+    } else {
+      return [];
```

Comment:
We won't need this change if we keep the QueryTreeViewItem-Type as is (and all other of these checks)